kustomize/commands/edit/set/configmap.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 The Kubernetes Authors.
```

Comment:
All other filenames in the `kustomize/commands/edit/set` directory begin with `set`. Would it be more consistent to name this file `setconfigmap.go`?

Reply:
Yes, that's currently following the same standard as the `add` folder. I plan to open a subsequent PR to change these names so they are all standardized.